### ISSUE TITLE
many eslint warnings

### ISSUE THREAD

**Issue by lowsky** [jscraftcamp/site-rewrite#1]
some are related to a11y  (like  jsx-a11y/anchor-has-content) or html struture (like react/jsx-no-target-blank):
```
⠀
Module Warning (from ./node_modules/eslint-loader/index.js):

/Users/robert.hostlowsky/github/JSCraftCamp-website/src/pages/sponsoring.jsx
  108:13  warning  Anchors must have content and the content must be accessible by a screen reader                                                                                                                                                                                                                                                                                          jsx-a11y/anchor-has-content
jsx-a11y/anchor-is-valid
  127:23  warning  Emojis should be wrapped in <span>, have role="img", and have an accessible description with aria-label or aria-labelledby


Module Warning (from ./node_modules/eslint-loader/index.js):

/Users/robert.hostlowsky/github/JSCraftCamp-website/src/pages/info2018.jsx
no-unused-vars
   54:46  warning  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images                                                                                                                                                                                                                                                                jsx-a11y/alt-text
   95:28  warning  Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener                                                                                                                                                                                                                                             react/jsx-no-target-blank
  105:28  warning  Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener                                           

```
